packages/devextreme/js/renovation/ui/scheduler/appointment_edit_form/__tests__/popup_config.test.ts

```diff
@@ -178,4 +180,16 @@ describe('API', () => {
       });
     });
   });
+
+  describe('popup dialogs', () => {
```

Comment:
Let's add the next test cases for these functions:
- let's check that the passed ```location``` argument exists in the ```result```
- let's check that the passed ```options``` argument exists in the ```result```